assets/runtime/functions

```diff
@@ -2153,6 +2154,9 @@ if [[ ${GITLAB_PAGES_ACCESS_CONTROL} == true ]]; then
       GITLAB_PAGES_ACCESS_REDIRECT_URI \
       GITLAB_PAGES_ACCESS_SECRET \
       GITLAB_PAGES_ACCESS_CONTROL_SERVER \
+      GITLAB_PAGES_NAMESPACE_IN_PATH \
+      GITLAB_PAGES_LOG_VERBOSE \
+      GITLAB_PAGES_NAMESPACE_IN_PATH \
```

Comment:
```suggestion
```
The argument `GITLAB_PAGES_NAMESPACE_IN_PATH` is specified multiple times to the same update_template. (I don't think this is harmful.)